src/modules/entidad_territorial/entidad_territorial.controller.ts

```
@@ -1,13 +1,13 @@
-    Post,
-    Body,
-    Param,
-    Patch,
-    Delete,
-    UsePipes,
-    UseGuards,
-    Controller,
+  Get,
+  Post,
+  Body,
+  Param,
```

ESLint

[error] 'Param' is defined but never used.

(@typescript-eslint/no-unused-vars)